src/plugins/condition/ConditionManager.js

```diff
@@ -56,20 +56,31 @@ export default class ConditionManager extends EventEmitter {
     );
   }
 
-  subscribeToTelemetry(endpoint) {
-    const id = this.openmct.objects.makeKeyString(endpoint.identifier);
-    if (this.subscriptions[id]) {
-      console.log('subscription already exists');
+  async requestLatestValue(endpoint) {
+    const options = {
+      size: 1,
+      strategy: 'latest'
+    };
+    const latestData = await this.openmct.telemetry.request(endpoint, options);
+    this.telemetryReceived(endpoint, latestData[0]);
```

Comment:
should we check to see that latestData is not falsy or empty array first? or do we want it to fail fast here?

Reply:
heuristically, i'm inclined to let it fail fast here

Reply:
If a condition is expected then handle it, if it's unexpected fail fast to help developers fix it. An empty array being returned from a telemetry provided is valid and expected. A falsy response is not.